src/includes/Common-Include.hs

```diff
@@ -99,6 +99,29 @@ hideCursor, showCursor :: IO ()
 hideCursor = hHideCursor stdout
 showCursor = hShowCursor stdout
 
+hUseAlternateScreenBuffer
+  :: Handle
+  -> IO ()
+
+hUseNormalScreenBuffer
+  :: Handle
+  -> IO ()
+
+-- | Use the Alternate Screen Buffer. If currently using the Normal Screen
+-- Buffer, it will save the cursor position and switch to the Alternate Screen
+-- Buffer. It will always clear the Alternate Screen Buffer. The Alternate
+-- Screen Buffer has no scroll back facility.
+useAlternateScreenBuffer
```

Comment:
It's worth to mention that it's application's responsbility to switch back to the normal screen buffer, even if its execution was interrupted abnormally.

Reply:
Good idea. I've now added something.